pkgs/tools/system/pciutils/default.nix

```diff
@@ -11,7 +12,7 @@ stdenv.mkDerivation rec {
     sha256 = "1ss0rnfsx8gvqjxaji4mvbhf9xyih4cadmgadbwwv8mnx1xvjh4x";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config hwdata ];
```

Comment:
The dependency here is useless.  (and technically it shouldn't be a native but a normal dependency)

Reply:
Ah right. I agree it's unnecessary in nativeBuildInputs, as the compilation process doesn't require hwdata. However, it shouldn't be added as a normal dependency, right? Else we draw in the full closure of hwdata which we don't require. We need just one file from hwdata.

Reply:
It wouldn't add a runtime dependency; those are computed by scanning the results.  But we don't need to add it anywhere.  It will become a build-time dependency just because it's used in some of the phases.

Reply:
Oh, I see! Thank you for pointing that out, didn't know that. Fixed in the new force-push.